addons/cargo/functions/fnc_startUnload.sqf

```diff
@@ -13,19 +13,20 @@
 
 #include "script_component.hpp"
 
-private ["_display", "_loaded", "_ctrl", "_selected", "_item"];
+private ["_display", "_loaded", "_loadedCount", "_ctrl", "_selected", "_item"];
 disableSerialization;
 _display = uiNamespace getvariable QGVAR(menuDisplay);
 if (isnil "_display") exitwith {};
 
 _loaded = GVAR(interactionVehicle) getvariable [QGVAR(loaded), []];
-if (count _loaded == 0) exitwith {};
+_loadedCount = count _loaded;
+if (_loadedCount == 0) exitwith {};
```

Comment:
you use count 3 time and this reduce it to one(and you dont change something at Loaded)